csvusingiterators.py

```
@@ -14,7 +14,7 @@ def __next__(self):
         except:
             StopIteration

```

@Devanshi-Sharma12 please raise this error